client/app/telecom/telephony/line/phone/codec/telecom-telephony-line-phone-codec.controller.js

```diff
@@ -124,11 +124,11 @@ angular.module("managerApp").controller("TelecomTelephonyLinePhoneCodecCtrl", fu
             return ["sip", "mgcp"].indexOf(service.featureType) > -1;
         });
 
-        return voipLinePhone.fetchAll().then((voipLinePhones) => {
-            return _.filter(filteredServices, function (service) {
+        return voipLinePhone.fetchAll().then((voipLinePhones) =>
+            _.filter(filteredServices, function (service) {
                 return _.some(voipLinePhones, { serviceName: service.serviceName, billingAccount: service.billingAccount });
-            });
-        });
+            })
+        );
     };
```

Comment:
fetchAll must returns result of _.filter() function or services won't be filtered